dataplane/apigen/ccgen/ccgen.go

```diff
@@ -668,8 +668,11 @@ return msg;
 		LOG(ERROR) << status.error_message();
 		return SAI_STATUS_FAILURE;
 	}
+	if (object_count != resp.resps().size()) {
```

Comment:
Wonder what can cause this situation to happen?

Reply:
if there's a bug in the server and it doesn't return the correct number of objects.